cc_corpus/index.py

```diff
@@ -52,7 +52,31 @@ def read_cluster_idx(cluster_idx: Path) -> Generator[Cluster]:
             yield Cluster.from_line(line)
 
 
-def find_tld_in_index(tld: str, cluster_idx: Path) -> list[Cluster]:
+def compare_inverse_list_urls(query_url, other_url):
+    """
+    Compares urls, given as inverted lists (e.g. ['hu','elte'] for elte.hu)
+
+    If the two are identical, or if the other url is the subdomain of the query
+    it returns 0
+    If  it is the query url which is the subdomain of the other, or
+    the other url is alphabetically (per domain element) before the query, it
+    returns -1
+    If the other url is alphabetically (per domain element) after the query, it
+    returns 1
+    """
+    for query_element, other_element in zip(query_url, other_url):
+        if query_element > other_element:
+            return -1
```

Comment:
I was thinking about which is the correct way... 

Instead of swapping the arguments, I swapped the return values (and changed the code which uses that value).